src/types/decodeData.ts

```diff
@@ -15,13 +15,14 @@ export interface DecodeDataInput extends DataInput {
 }
 
 export interface DecodeDataOutput {
-  value: string | string[] | URLDataWithHash;
+  value: string | string[] | URLDataWithHash | null;
   name: string;
   key: string;
 }
 
 export interface FetchDataOutput {
   value:
+    | null
```

Comment:
just to clarify why do we return null here now?

Reply:
We need the null for the jests. We were not typechecking fully previously. We use null in places although it should really be `value?` but I didn't think that makes sense to mix into this PR.

Reply:
nice thanks for clarifying